frontend/device/input.lua

```diff
@@ -658,10 +660,8 @@ function Input:handleTouchEv(ev)
         end
     elseif ev.type == C.EV_SYN then
         if ev.code == C.SYN_REPORT then
-            -- Promote our event's time table to a real TimeVal
-            setmetatable(ev.time, TimeVal)
             for _, MTSlot in ipairs(self.MTSlots) do
-                self:setMtSlot(MTSlot.slot, "timev", ev.time)
+                self:setMtSlot(MTSlot.slot, "timev", time.tv(ev.time))
```

Comment:
time.tv() returns a TimeVal-like table I guess ? So, `time.timeval(ev.time)` , something less abbreviated than `tv()` ?

Reply:
Correct. :+1:

Reply:
time.s, time.ms, time.us, time.timeval do all the same: convwrt the given value to an fts encoded time.